components/smee/staging/deployment.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gosmee
+  labels:
+    app: gosmee
+  annotations:
+    ignore-check.kube-linter.io/liveness-port: "gosmee probes sidecar port"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gosmee
+  template:
+    metadata:
+      labels:
+        app: gosmee
+    spec:
+      containers:
+        - image: "ghcr.io/chmouel/gosmee:v0.20.2"
+          imagePullPolicy: Always
+          name: gosmee
+          args: ["server", "--address", "0.0.0.0"]
+          ports:
+            - name: "gosmee-http"
+              containerPort: 3333
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 9100
+            initialDelaySeconds: 30
+            periodSeconds: 30
+            timeoutSeconds: 25 # Must be > the healthz handler's timeout
+            failureThreshold: 2
+          initialDelaySeconds: 30
+          periodSeconds: 5
+          timeoutSeconds: 2
+          failureThreshold: 3
+          securityContext:
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+          resources:
+            limits:
+              cpu: 1
+              memory: 256Mi
+            requests:
+              cpu: 1
+              memory: 256Mi
+        - image: "ghcr.io/chmouel/gosmee:v0.26.1"
+          imagePullPolicy: Always
+          name: gosmee
+          args:
+            - "client"
+            - "http://localhost:3333/smeesvrmonit"
+            - "http://localhost:8080"
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - "ALL"
+            seccompProfile:
+              type: RuntimeDefault
+            readOnlyRootFilesystem: true
+            runAsNonRoot: true
+          resources:
+            limits:
+              cpu: 1
+              memory: 750Mi
+            requests:
+              cpu: 1
+              memory: 750Mi
```

Comment:
This client is going to serve 2 events once every 30 seconds. I think we can try using less resources.

Reply:
In konflux-ci/konflux-ci we have this
```
          resources:
            limits:
              cpu: 100m
              memory: 32Mi
            requests:
              cpu: 10m
              memory: 32Mi
```
Probably we can do the same, but make requests equal limits.